pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -165,6 +162,11 @@ stdenv.mkDerivation (finalAttrs: {
     # Required for GSettings schemas at runtime.
     # Will be picked up by wrapGAppsHook4.
     gsettings-desktop-schemas
+  ] ++ lib.optionals stdenv.hostPlatform.isDarwin [
+    (darwinMinVersionHook "15.0")
```

Comment:
wait a sec -- I wanted 10.15 here.

Reply:
Uh, whoops. Do you mind opening a PR to fix? Otherwise I’ll get to it tomorrow.

(Finally an aggressive macOS version support policy!)

Reply:
https://github.com/NixOS/nixpkgs/pull/351053